app/components/agenda/agenda-list/list-item/template.hbs

```diff
@@ -24,6 +24,13 @@
       {{/if}}
     {{/if}}
   {{/if}}
+
+  {{#if (not isEditingOverview)}}
+  {{#if isEditor}}
+    <span class="vl-u-text--bold ">{{ t "remark-title" }}: </span> {{await agendaitem.explanation}}
+  {{/if}}
```

Comment:
Maybe add a check if agendaitem.explanation is '', don't show the span.
Also explanation is a string, not a relation which shouldn't be awaited.

Reply:
Also check https://kanselarij.atlassian.net/browse/KAS-1154. 
Add some margin-top or check with @rafael-mindreau to make it a little bit less dominant.
Maybe something with the font and the positioning.

Reply:
Aangepast. Als Raf terug is zullen we de css is bezien.